README.md

```diff
@@ -1,4 +1,12 @@
+<p align="center">
+  <img src="https://katanox.com/images/logo-white.png" />
+</p>
+
 # Katanox PHP SDK
+![test workflow](https://github.com/katanox/katanox-php/actions/workflows/test.yaml/badge.svg)
+![StyleCI](https://github.styleci.io/repos/397629408/shield?branch=master)
```

Comment:
is this working?

Reply:
Yes, we just need to adjust a couple of files to fully implement PSR-12